Makefile

```diff
@@ -230,6 +230,12 @@ run-playwright:
 ## Run end-to-end tests with playwright (see run-playwright)
 e2e: run-playwright
 
+## Run end-to-end tests and list failed snapshot test image files.
+e2e-diff: PLAYWRIGHT_ARGS = --reporter json
+e2e-diff:
+	$(PLAYWRIGHT_CMD) | \
```

Comment:
annoyingly, when this runs in docker (and perhaps locally in some circumstances), the `npm ci` command produces output that is not json and that breaks.

You will need to add `> /dev/null` at the end of the `npm --prefix e2e ci` command.

I also added `--env NPM_CONFIG_UPDATE_NOTIFIER=false` on the `PLAYWRIGHT_CMD_DOCKER` command after the other `--env` to silence the silly update notification - silly because we cannot update npm in the docker image. This output did not interfere with the json output, but it is annoying to see so nice to be rid of. Can you add these?